sechub-testframework/src/main/java/com/mercedesbenz/sechub/test/TestUtil.java

```diff
@@ -165,7 +165,7 @@ public static Path createTempDirectoryInBuildFolder(String dirName, FileAttribut
      *
      * @param explicitFileName the EXACT file name to use.
      *
-     * @return file
+     * @return filePath
```

Comment:
Please change back to "file" - Path is the java.nio representation of an file or folder..